datasets/es/parliament/crawler.py

```diff
@@ -0,0 +1,167 @@
+import string
+from urllib.parse import urlparse, parse_qs
+
+from zavod import Context, helpers as h
+from zavod.stateful.positions import categorise
+
+
+SENATORS_URL = "https://www.senado.es/web/composicionorganizacion/senadores/composicionsenado/consultaordenalfabetico/index.html"
+DEPUTIES_URL = "https://www.congreso.es/en/busqueda-de-diputados?p_p_id=diputadomodule&p_p_lifecycle=2&p_p_state=normal&p_p_mode=view&p_p_resource_id=searchDiputados&p_p_cacheability=cacheLevelPage"
+FORM_DATA = {
+    "_diputadomodule_idLegislatura": "15",
+    "_diputadomodule_genero": "0",
+    "_diputadomodule_grupo": "all",
+    "_diputadomodule_tipo": "0",
+    "_diputadomodule_formacion": "all",
+    "_diputadomodule_filtroProvincias": "[]",
+}
+IGNORE = ["constituency_id", "legislative_term_id", "gender"]
+
+
+def rename_headers(context, entry):
+    result = {}
+    for old_key, value in entry.items():
+        new_key = context.lookup_value("columns", old_key)
+        if new_key is None:
+            context.log.warning("Unknown column title", column=old_key)
+            new_key = old_key
+        result[new_key] = value
+    return result
+
+
+def emit_pep_position(
+    context,
+    pep,
+    title,
+    start_date,
+    end_date,
+    subnational_area,
+    is_pep,
+):
+    pep.add("position", title)
+    position = h.make_position(
+        context,
+        name=title,
+        country="es",
+        subnational_area=subnational_area,
+        lang="spa",
+        topics=["gov.legislative", "gov.national"],
+    )
+    categorisation = categorise(context, position, is_pep=is_pep)
+    occupancy = h.make_occupancy(
+        context,
+        pep,
+        position,
+        start_date=start_date,
+        end_date=end_date,
+        categorisation=categorisation,
+    )
+    if occupancy:
+        context.emit(occupancy)
+        context.emit(position)
+        context.emit(pep)
+
+
+def crawl_deputy(context, item):
+    name = item.pop("full_name")
+    id = item.pop("parliament_member_id")
+    party = item.pop("party")
+
+    pep = context.make("Person")
+    pep.id = context.make_id(id, name, party)
+    h.apply_name(
+        pep,
+        full=name,
+        first_name=item.pop("first_name"),
+        last_name=item.pop("last_name"),
+    )
+    pep.add("political", party)
+    pep.add("political", item.pop("parliamentary_group"))
+    pep.add("topics", "role.pep")
```

Comment:
you can leave it to make_occupancy to apply the pep topic then it's automatically part of the 

```
if position.is_pep:
    make_occupancy(...)
```

pattern